extensions/cornerstone-dicom-sr/src/utils/hydrateStructuredReport.ts

```diff
@@ -215,10 +215,14 @@ export default function hydrateStructuredReport(
         dataSource
       );
 
-      commandsManager.runCommand('updateMeasurement', {
-        uid: newAnnotationUID,
-        code: annotation.data.finding,
-      });
+      const style = annotation.data.finding?.style || annotation.data.site?.style;
```

Comment:
This is fixing a bug on rehydrating in the basic-test mode that was being seen when testing the rest of the layout.